spec/queries/keywords_query_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe(KeywordsQuery) do
+  describe '#call' do
+    context 'given NO keywords' do
+      it 'returns an empty list' do
+        Keyword.delete_all
+
+        keywords = described_class.new.call
+
+        expect(keywords).to(be_empty)
+      end
+    end
+
+    context 'given 5 keywords' do
+      it 'returns 5 keywords' do
+        Keyword.delete_all
```

Comment:
This steps might be overkill as `rspec` reset the database before every test or place db operations in transactions that are rolled back between tests 💡

Reply:
I was wondering if that violates this guideline https://nimblehq.co/compass/development/code-conventions/ruby/rspec/#incidental-state 🤔

Reply:
No, because Rails handles it for us :-)